Appsilon/mbaza
# Override predictions of selected observations at once

### Changes
Closes #300 

-  add check icon button to the bottom right corner
-  handle card appearance change on the icon button click
-  activate multiselection mode on icon button click
-  change appearance of box header on icon button click
-  create ObservationsHeader component
-  handle multiselect mode cancellation
-  display how many cards selected
-  add dropdown element to the right side of box header
-  update box header styling
-  add Update button overriding predictions in selected cards
-  add minor styling improvements
-  refactor
-  handle updating overrides for all selected cards

### How to test

- check if overriding selected cards / single cards works correctly
- check code
